romancal/regtest/regtestdata.py

```diff
@@ -25,6 +25,9 @@
 ARTIFACTORY_API_KEY_FILE = "/eng/ssb2/keys/svc_rodata.key"
 
 
+TIMEOUT = 30  # seconds
```

Comment:
The bandit security audits have been updated. One new thing it catches is that all `requests.get` should include a `timeout` argument. I have guessed 30s though I don't know anything about the server we are making a request from.